src/hexo/themes/sonarwhal/source/core/css/scan-results.css

```diff
@@ -16,6 +16,32 @@
     padding: 1rem 2rem;
 }
 
+.scan-overview__header p:first-child {
+    margin-right: 5rem;
+    margin-bottom: 1rem;
+}
+
+.scan-overview__status {
+    margin: 0;
+    text-transform: capitalize;
+}
+
+#status-analyzing {
```

Comment:
I don't think this should be an ID. When we change the status to finished we are forced to change the ID otherwise it will still be animated. I think it would be better to be a class and remove it. @ststimac, @qzhou1607 what do you think?